src/_companies/europe/france/auvergne-rhone-alpes/etaminstudio/index.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "Etamin Studio"
+
+continent: "Europe"
+country: "France"
+region: "Auvergne-Rhône-Alpes"
+city: "Clermont-Ferrand"
+
+website: https://etamin.studio
+logo: logo.png
+
+stack:
+  - "rails"
+  - "postgres"
+  - "stimulus"
+
+created_at: 2023-03-20 09:47:00
+---
+
+We are passionate developers and designers, creators of quality web and mobile experiences.
```

Comment:
Could you change this to be in the third person? I'd like to keep it consistent for all company descriptions :)

Reply:
Will do!